### PR TITLE
Changed input form setup and JS logic to support. Added auto scroll t…

### PR DESCRIPTION
**Pull request by JonPointer**
…o bottom of messages

### CODE REVIEW

File: public/js/messages.js [hunk 1]
```diff
@@ -35,76 +37,131 @@ userDisplay = (objButton) => {
   //   });
 };
 
+var msgFromUser = "";
 
 var userID = localStorage.getItem("userID");
 var usersName = localStorage.getItem("usersName");
```
Comment (jamesosull16) [inline public/js/messages.js]: can we change these 'var' to 'const'?

**Review by jamesosull16 — DISMISSED** [review]
Looks good and works locally.  Just the change from var to const in the messages.js file and I think we are good.  

One other thing I noticed is that the "Refresh Message" button is not actually refreshing the messages as much as the user list.  So in a new PR I think we can change that to "Refresh Users".

**Comment by jamesosull16** [comment]
Another thing I noticed is that if I am to enter my name as 'James' it saves it as a different user than if I were to enter 'james'.  We're not validating the input in that regard.  If we're going to be slacking out the link for people to use during our demo, I don't want them entering their name with a capitol and lowercase letter and calling it out you know. Something we can discuss tonight

**Review by kaiasahite — APPROVED** [review]
awesome!

**Review by kaiasahite — APPROVED** [review]

**Review by kaiasahite — APPROVED** [review]

**Pull request merged**